scipy/optimize/_qap.py

```diff
@@ -0,0 +1,418 @@
+import numpy as np
+import math
+from . import linear_sum_assignment, minimize_scalar, OptimizeResult
+
+
+def quadratic_assignment(
+    cost_matrix,
+    dist_matrix,
+    seed=None,
+    maximize=False,
+    n_init=1,
+    init_method="barycenter",
+    max_iter=30,
+    shuffle_input=True,
+    eps=0.05,
+):
+    """
+    Solve the quadratic assignment problem
```

Comment:
```suggestion
    Solve the quadratic assignment problem.
```

I think this is showing up here?
![image](https://user-images.githubusercontent.com/6570539/82103893-a8095c00-96c9-11ea-9b56-1ecd1126d28d.png)